model/src/main/kotlin/vulnerabilities/VulnerabilityReference.kt

```diff
@@ -57,8 +57,9 @@ data class VulnerabilityReference(
     val score: Float?,
 
     /**
-     * The full CVSS vector, if any, this reference assigns to the belonging vulnerability. Note that while the vector
-     * usually contains the [scoringSystem], that is not the case for e.g. CVSS version 2.
+     * In case of CVSS, this is the full vector, if any, this reference assigns to the belonging vulnerability. Note
+     * that while the vector usually contains the [scoringSystem] as a prefix, that is not the case for CVSS version 2.
+     * In case of EPSS, this is the precentile which provides a measure of probability relative to all other scores.
```

Comment:
* The first sentence is hard to read, consider: "In case of VCSS, this is the full vector this reference assigns to the belonging vulnerability, if available."
* Typo in "percentile".